packages/core/root/core-root-server-internal/src/bootstrap.ts

```diff
@@ -79,6 +79,9 @@ export async function bootstrap({ configs, cliArgs, applyConfigOverrides }: Boot
 
   const root = new Root(rawConfigService, env, onRootShutdown);
 
+  // eslint-disable-next-line no-console
+  console.log('Configurations parsed in this order: ' + env.configs.join(', '));
```

Comment:
> For the tests, I needed to output something to the logs.

I would expect this to be replaceable with unit tests of the proper sub component(s).

Reply:
it's a good idea, now that it's one component, I could do a unit test for it